src/app/components/SubjectHeading/SubjectHeadingShow.jsx

```diff
@@ -155,13 +155,15 @@ class SubjectHeadingShow extends React.Component {
                 showId={uuid}
                 keyId="context"
                 container="context"
+                seeMoreLinkUrl={linkUrl}
+                seeMoreText="See More in Subject Headings Index"
```

Comment:
Did this approach get merged in while I was out? I still think these props can and should be determined by `AdditionalSubjectHeadingsButton` so we do not have two extra props being passed for a detail.

Reply:
Yeah I see the argument for keeping the props small, on the other hand I still think that it is more modular if the `AdditionalSubjectHeadingsButton` doesn't need to know about where it is being rendered to work properly. If you want this could be a good point to discuss in the Wednesday meeting.

Reply:
Sure. There could a more modular component, with some wrapper component still handling the logic for how to render that more modular component.